### DISCUSSION TITLE
Why the LL-HLS using separate tracks?

### DISCUSSION THREAD

**Discussion by xiaozhihong** [OvenMediaLabs/OvenMediaEngine#1294] [Help] [answered]
Why does the LL-HLS implement using separate tracks? I can not find any specifics about this.

**Comment by getroot** [comment, marked as answer]
Not required. I have separated tracks to support ABR efficiently. Also, I decided it would be better to reduce the buffer for A/V merging and let the player handle it.

**Reply by xiaozhihong** [reply]
Thanks for your reply.

**Reply by getroot** [reply]
FYI, Apple recommends this as a "SHOULD" for HLS Multivariant (ABR).

https://developer.apple.com/documentation/http-live-streaming/hls-authoring-specification-for-apple-devices#Multivariant-Playlist

**Reply by xiaozhihong** [reply]
Yes, but no mention LL-HLS should use ABR.

**Reply by getroot** [reply]
Yes, LL-HLS does not require separated A/V tracks.
